src/components/ViewRoutingSlip/PaymentInformation.vue

```diff
@@ -36,12 +36,21 @@
                   }}</v-icon>
                 </v-btn>
               </v-col>
+              <v-col class="font-weight-bold d-flex justify-end" v-if="!isEditable && isExpanded && routingSlip && routingSlip.payments && $route.meta.allowedRoles.includes(Role.FAS_CORRECTION)">
+                <span
+                    class="primary--text cursor-pointer"
+                    @click="adjustRoutingSlipStatus"
+                  >
+                    <v-icon color="primary" size="20"> mdi-pencil</v-icon>
+                    Adjust Routing Slip
+                </span>
+              </v-col>
             </v-row>
             <v-row no-gutters v-if="isExpanded && routingSlip && routingSlip.payments" class="mb-10">
               <v-expand-transition>
                 <v-col cols="11">
-                  <review-routing-slip-cheque-payment data-test="review-routing-slip-cheque-payment" v-if="isPaymentCheque" :chequePayment="routingSlip.payments" :isAmountPaidInUsd="isRoutingSlipPaidInUsd"/>
-                  <review-routing-slip-cash-payment data-test="review-routing-slip-cash-payment" v-else :cashPayment="routingSlip.payments[0]" :isAmountPaidInUsd="isRoutingSlipPaidInUsd"/>
+                  <review-routing-slip-cheque-payment data-test="review-routing-slip-cheque-payment" v-if="isPaymentCheque" :chequePayment="routingSlip.payments" :isAmountPaidInUsd="isRoutingSlipPaidInUsd" :isEditable="isEditable"/>
```

Comment:
This should be consistent with review-routing-slip-cheque-payment, with the attributes on multiple lines I think.